Mathlib/MeasureTheory/Measure/AddContent.lean

```diff
@@ -24,15 +24,15 @@ A `Content` is in particular an `AddContent` on the set of compact sets.
 
 ## Main statements
 
-Let `m` be an `AddContent C`. If `C` is a set semi-ring (`IsSetSemiring C`) we have the properties
+Let `m` be an `AddContent C`. If `C` is a set semiring (`IsSetSemiring C`) we have the properties
```

Comment:
[Wikipedia](https://en.wikipedia.org/wiki/Ring_of_sets) uses a mix of both ... By Googling `semi-ring measure` it seems semi-ring is somewhat common in the measure theory context.